testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/response/resource/PublisherResponseMessageBodyWriter.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.resteasy.test.response.resource;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import java.nio.charset.StandardCharsets;
+
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.MessageBodyWriter;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+@Produces("abc/xyz")
+public class PublisherResponseMessageBodyWriter implements MessageBodyWriter<PublisherResponseTestClass> {
```

Comment:
Sorry, I'm not getting what this is used for / what is tested through this (together with the PublisherResponseTestClass). What am I missing?

Reply:
Totally a mistake, well spotted I'll remove it.

Reply:
Thanks!